web/src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -247,6 +247,10 @@ export class FeedbackBoxContainer extends React.Component<PropsType, StateType>
     }
   }
 
+  handleSubmit = () => {
+    this.submitFeedback()
+  }
```

Comment:
So whats this one about? Why do you need to "wrap" `submitFeedback`?

Reply:
`submitFeedback` returns a promise while `handleSubmit` does not. So another flow issue.